lib/paper_trail/attribute_serializers/attribute_serializer.rb

```diff
@@ -0,0 +1,22 @@
+require "paper_trail/attribute_serializers/array_serializer"
+
+module PaperTrail
+  module AttributeSerializers
+    # Values returned by some Active Record serializers are
+    # not suited for JSON encoding. This factory
+    # replaces certain default Active Record serializers
+    # with custom PaperTrail ones.
+    module AttributeSerializer
```

Comment:
I see what you were going for with this name. `AttributeSerializer.for` does read nicely, but I'd think I'd prefer `AttributeSerializerFactory.for`. It doesn't read as well, but you can tell the purpose of the module from its name alone.